views/install.sh.erb

```diff
@@ -171,6 +171,13 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
+  platform=$ID
+  if test "x$platform" = "x"; then
+    platform="linux"
+  fi
```

Comment:
Yes, you're not adhering to that API, you're adhering to omnitruck's API which closely models ohai's platform.

Which may be a larger issue, you probably really need a case statement here to translate from os-release to ohai platform name.